src/SignalR/server/Core/src/HubConnectionHandler.cs

```diff
@@ -133,7 +152,14 @@ private async Task RunHubAsync(HubConnectionContext connection)
         {
             try
             {
-                await _dispatcher.OnConnectedAsync(connection);
+                if (!await _dispatcher.OnConnectedAsync(connection))
```

Comment:
Unrelated to the PR?

Reply:
Do we want to detect if Hub.OnconnectedAsync threw or was never called and a hub filter swallowed the exception?

Reply:
Continue as normal. This is an advanced API.

If a filter doesn't call OnconnectedAsync or swallows the exception, that's the filter author's choice. If the filter author want to force a close in OnConnectedAsync, the filter can throw.